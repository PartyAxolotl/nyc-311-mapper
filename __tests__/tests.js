import renderer from 'react-test-renderer';
import MainPage from '../client/components/Pages/MainPage'
import MapPage from '../client/components/Pages/MapPage'
import { GoogleMap } from '@react-google-maps/api';

describe('App', () => {
  it('renders correctly', () => {
    const pageRender = renderer.create(<MainPage />).toJSON();
    expect(pageRender).toMatchSnapshot()
  })
})

describe('MainPage', () => {
  it('renders correctly', () => {
    const partySpots = [{ lat: 40.730610, lng: -73.935242 }]
    const mapRender = renderer.create(<MapPage partySpots={partySpots} />).toJSON();
    expect(mapRender).toMatchSnapshot()
  })
})

describe('MapPage', (props) => {
  it('renders correctly', () => {
    const googleRender = renderer.create(<GoogleMap />).toJSON();
    expect(googleRender).toMatchSnapshot()
  })
})

