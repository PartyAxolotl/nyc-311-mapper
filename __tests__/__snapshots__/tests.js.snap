// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
[
  <div
    id="pfHeader"
  >
    <h1
      id="mainPage"
    >
      PartyFinder
    </h1>
  </div>,
  <form
    onSubmit={[Function]}
  >
    <div
      id="formLine"
    >
      <label
        htmlFor="boroughBox"
      >
        Show me party spots in 
      </label>
      <select
        className="dropDownBoxes"
        id="boroughSelect"
        onChange={[Function]}
      >
        <option
          value="MANHATTAN"
        >
          Manhattan
        </option>
        <option
          value="BROOKLYN"
        >
          Brooklyn
        </option>
        <option
          value="BRONX"
        >
          The Bronx
        </option>
        <option
          value="QUEENS"
        >
          Queens
        </option>
        <option
          value="STATENISLAND"
        >
          Staten Island
        </option>
        <option
          value="ALL"
        >
          ALL OF NYC!!!
        </option>
      </select>
      <label
        htmlFor="daySelect"
      >
         on 
      </label>
      <select
        className="dropDownBoxes"
        id="daySelect"
        onChange={[Function]}
      >
        <option
          value="1"
        >
          Mondays.
        </option>
        <option
          value="2"
        >
          Tuesdays.
        </option>
        <option
          value="3"
        >
          Wednesdays.
        </option>
        <option
          value="4"
        >
          Thursdays.
        </option>
        <option
          value="5"
        >
          FRIDAYS!
        </option>
        <option
          value="6"
        >
          SATURDAYS!
        </option>
        <option
          value="0"
        >
          Sundays?
        </option>
      </select>
      <input
        id="goButton"
        type="submit"
        value=" GO "
      />
    </div>
  </form>,
  <div
    id="overlay"
  />,
  <div
    id="hey"
  >
    <div
      id="layer-up"
    />
  </div>,
  <div
    id="layer-0"
  >
    <div
      id="layer-1"
    >
      <div
        id="layer-2"
      >
        <div
          id="lines"
        >
          <div
            id="layer-corner"
          />
        </div>
      </div>
    </div>
  </div>,
  <div
    id="mtnZZZ"
  />,
]
`;

exports[`MainPage renders correctly 1`] = `
<div
  className="App"
>
  <h1>
    Loading...
  </h1>
</div>
`;

exports[`MapPage renders correctly 1`] = `<div />`;
